core-web/libs/sdk/react/src/lib/components/DotEditableText/DotEditableText.tsx

```diff
@@ -5,7 +5,8 @@ import {
     isInsideEditor as isInsideEditorFn,
     postMessageToEditor,
     CUSTOMER_ACTIONS,
-    DotCmsClient
+    DotCmsClient,
+    NOTIFY_CUSTOMER
```

Comment:
We can't keep adding internal things exposed to `@dotcms/client` we can either create another lib (ughhh) or move our internal things out of the index, like `@dotmcs/client/internal/` or something like that. Check what other libs do here.

Reply:
This can be in the Types lib, when we create it